pkg/operator/controller/names.go

```diff
@@ -15,6 +15,12 @@ const (
 	// CA certificate in this namespace.
 	GlobalMachineSpecifiedConfigNamespace = "openshift-config-managed"
 
+	// ClusterScopedNamespace is an empty string representing a cluster scoped namespace.
+	ClusterScopedNamespace = ""
+
+	// OpenShiftConfigNamespace is the namespace for configuring OpenShift.
+	OpenShiftConfigNamespace = "openshift-config"
```

Comment:
over-abstraction. This const just makes it harder to read.

Reply:
I was asked by the networking-team to include the constants [here](https://github.com/openshift/cluster-ingress-operator/pull/552#discussion_r578565736).

Reply:
@sgreene570 what are your thoughts?

Reply:
If you feel strongly about moving or removing the const I won't object. Just made sense to me to include the openshift config namespace as a const with all of the other namespace consts the operator may need.